coreos-base/oem-gce/files/units/oem-gce.service

```diff
@@ -19,6 +19,8 @@ ExecStart=/usr/bin/rkt run \
     --volume=etc,kind=host,source=/etc,readOnly=false \
     --volume=home,kind=host,source=/home,readOnly=false \
     --volume=runsystemd,kind=host,source=/run/systemd,readOnly=false \
+    --volume=nsswitch,kind=host,source=/usr/share/google-oslogin/nsswitch.conf \
+    --mount=volume=nsswitch,target=/usr/share/google-oslogin/nsswitch.conf \
```

Comment:
Should it be maybe with `,readOnly=true`?

Reply:
Can add it, but the container is actually supposed to have write access to `/etc` anyway.

Reply:
Done